### ISSUE TITLE
Write request fails in Prometheus v2.54.0 (2024-08-09) 

### ISSUE THREAD

**Issue by 029A-h** [huksley/prometheus-remote-write#7]
In the recent version of Prometheus (since v2.54.0), write requests fail with a 415 HTTP status code (Unsupported Media Type) and the error message:
`expected application/x-protobuf as the first (media) part, got application/vnd.google.protobuf content-type`.

As observed in [write_handler.go:81](https://github.com/prometheus/prometheus/blob/3668194bd15fc15d85f0f283feaeddc38792db18/storage/remote/write_handler.go#L81C17-L81C36), the Content-Type header is validated against `application/x-protobuf`, which is different from the default header `application/vnd.google.protobuf` used in the [prometheus-remote-write](https://github.com/huksley/prometheus-remote-write/blob/89663f323e34f2896ca5e23933ee85c30541c646/index.js#L98) library.

Quick Fix:
Add the Content-Type header to the config object as follows:

```
Copy code
const config = {
  ...
  headers: {
    'Content-Type': 'application/x-protobuf'
  }
  ...
};
```

Suggested Solution:
Change the default content type in remote write requests to application/x-protobuf.

**Comment by huksley** [comment]
Yeah, it seems like 'application/vnd.google.protobuf' is a legacy thing, even 1.0 older spec requires 'application/x-protobuf' content type. I am glad there no other changes needed...

https://prometheus.io/docs/specs/remote_write_spec/

**Comment by huksley** [comment]
Fixed in v0.5.0

**Comment by huksley** [comment]
Sorry for taking this so long! Can you, @029A-h check this? I released new version v0.5.0 which fixes this.

Unfortunately I can't properly test this, since I am no longer use Prometheus a lot these days.

**Comment by 029A-h** [comment]
I've tested v0.5.0 with no headers passed in config and default ones work fine.
Tested against Prometheus v2.50.0.
Thanks for keeping this library alive!
